README.md

```diff
@@ -40,4 +40,13 @@ The data required to build the templates is in the /resources folder in the /tem
 
 When building the templates, the i18n Handlebars helper will use the data for the current tenant by iterating through the property items for the 'culture' that has been passed in via the model.
 
-## Documentation to be completed once module is in stable state.
+## Testing
+
+Footer links are integration tested, however must be manually run by the developer due to CORS issues with JSDOM and the Just Eat international websites.
+
+To tests the links:
+1. Open the `package.json` and change `testEnvironmentOptions.url` to the page you want to test
```

Comment:
Do you need to add a comment/step about removing the skip hook on the describe block?

Reply:
cheers, will add that

Reply:
Done 👍